src/module/add-like.js

```diff
@@ -0,0 +1,15 @@
+async function addLike(id) {
+  fetch('https://us-central1-involvement-api.cloudfunctions.net/capstoneApi/apps/tnE2k6P5BdZ2HCTjbd0V/likes', {
+    method: 'POST',
+    body: JSON.stringify({
+      item_id: id,
+      likes: 1,
```

Comment:
No need to add "likes:1" to POST, it will automatically add only1 like.

Reply:
Thank you for the feedback, it was very helpful.👍